tsconfig.json

```diff
@@ -1,20 +1,4 @@
 {
-  "compilerOptions": {
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx"
-  },
+  "extends": "./tsconfig.base.json",
```

Comment:
@AlicanC on this project as we don't have packages and stuff would be nice to keep tsconfig.json and tsconfig.base.json in a single file. But no big deal!

Reply:
We don't but we still need multiple configurations: one for files under `src`, which use `tsconfig.json` and another for the rest, for example for files under `script` or in repo root, which use `tsconfig.eslint.json`. Both of these configurations import `tsconfig.base.json`.